### DISCUSSION TITLE
Hopefully a simple question.

### DISCUSSION THREAD

**Discussion by Synghyn** [Jyers/Marlin#1718] [Q&A]
Updated to E3V2-BLTouch-3x3-v4.2.2-v2.0.1.  Was going through the new items in the menus and noticed the size of the printer is set at 250 height.  The Ender 3V2 is 350 height.  Is this just mislabeled, is there somewhere to set it correctly?  Will the Creality 350mm patch fix it?  
By the way, looks like a great version, thanks for the work Jyers!

Synghyn

**Comment by HAPSagan** [comment]
Technical Specifications
 
Ender-3 V2 FDM 3D Printer Properties
 
Modeling Technology: FDM（Fused Deposition Modeling）
 
Printing Size: 220x220x250mm
 
Filament: PLA/TPU/PETG
 
Working Mode: Online or SD card offline
 
Supported OS: MAC/WindowsXP/7/8/10
 
Filament Diameter: 1.75mm
 
Slicing Software: Simplify3d/Cura
 
Ender-3 V2 FDM 3D Printer Hardware
 
Machine Size : 475x470x620mm
 
Product Weight: 7.8KG
 
Package Weight: 9.6KG
 
Power Supply: Input AC 115V/230V; Output DC 24V 270W
 
Ender-3 V2 FDM 3D Printer Extruder Hardware
 
Layer Thickness: 0.1-0.4mm
 
Print Precision: ±0.1mm
 
Hotbed Temperature: ≤100°

**Comment by Cal-LF** [comment]
The measurement on my E3V2 is 250mm height to the print nozzle.

**Comment by Synghyn** [comment]
Would have been more helpful if I'd mentioned the 350 upgrade, my bad. 

Ender 3V2 upgrade ](https://user-images.githubusercontent.com/96601668/147365083-ef4b2946-2579-46ce-85dd-ed719c70397a.pn)

**Comment by PapsDizzy** [comment]
e3v2 is 250mm . or u have moddet it.

**Comment by Cal-LF** [comment]
The bed size can be changed in the config.h file and then compiled.

![size](https://user-images.githubusercontent.com/76499230/147367340-7aa314b9-7c50-4223-aeef-aa7e882ab6bf.jpg)

**Comment by Synghyn** [comment]
Papsdizzy, yes, like I mentioned in the post before yours I did do the modification from Creality to raise the print area 10 cm.
Cal-LF, I figured that was going to be one way to fix it, but I have had a bad time with compiling firmware in the past and would rather avoid it.  I have a patch from that mod to change the height.  I guess the better question to start with would have been would the upgrade provided for that mod be likely to fix it if applied to this firmware?

**Reply by Cal-LF** [reply]
@Synghyn I don't know how the firmware can be modified on the printer. The .bin file is the result of compiling. The file cannot be changed after compiling.

**Reply by Synghyn** [reply]
I'd assume it is a full firmware file with the height changed.  Honestly the firmware flashes so quickly now a days, it seems like a small patch.  
Ender-3 V2_Marlin2.0.8_HW_V4.2.2_SW_V2.0.8.4_ALT_Zh_CN_en_20210715.bin is the firmware update for that I think.  Not a hundred percent sure, I haven't labelled everything as well as I Probably should have.  But based on the ALT_Z in the middle I think that was the one.  Guess if I want the much better firmware that Jyers put out over Creality's I'll have to figure out compiling it.  I know it isn't that hard to do, just as I said earlier, I've had some issues with it before.  Which I suppose means I ought to dig in and learn it better.
